ALLPERMS isn't specified in POSIX so musl doesn't define it

diff --git a/src/FileInfo.cpp b/src/FileInfo.cpp
index 5cc198c..197d40c 100644
--- src/FileInfo.cpp
+++ src/FileInfo.cpp
@@ -21,6 +21,10 @@
 #include "Logger.h"
 #include "Exception.h"
 
+#ifndef ALLPERMS
+#   define ALLPERMS (S_ISUID|S_ISGID|S_ISVTX|S_IRWXU|S_IRWXG|S_IRWXO)
+#endif
+
 // Some file systems (NTFS seems to be among them) may handle block fragments
 // well. Don't report files as "sparse" files if the block size is only a few
 // bytes less than the byte size - it may be due to intelligent fragment

